tests/user_restricted/test_user_restricted.py

```diff
@@ -31,16 +33,30 @@ def test_retrieve_deprecated_url(self, headers_with_token):
 
         helpers.check_response_status_code(response, 404)
 
-    # @pytest.mark.smoke_test
+    @pytest.mark.smoke_test
     @pytest.mark.nhsd_apim_authorization(AUTH_HEALTHCARE_WORKER)
-    def test_retrieve_patient(self, headers_with_token):
+    @pytest.mark.skipif(ENVIRONMENT != 'int', reason="INT can only use pre-built test app")
+    def test_retrieve_patient_for_int(self,
+                                      apigee_environment,
+                                      nhsd_apim_config,
+                                      _test_app_credentials):
+        headers = helpers.get_headers(apigee_environment,
+                                      nhsd_apim_config,
+                                      _test_app_credentials,
+                                      AUTH_HEALTHCARE_WORKER)
+        self.retrieve_patient_and_assert(retrieve[0], headers)
 
+    @pytest.mark.nhsd_apim_authorization(AUTH_HEALTHCARE_WORKER)
+    def test_retrieve_patient_for_non_int(self, headers_with_token):
```

Comment:
Are only smoke tests run against int?

Reply:
I'm not convinced I'll keep the skipif marker with later refactoring. But didn't want to make a big upheaval to only do it differently once I do the larger refactoring for this ticket.

Reply:
Agree, it's/they're a bit naff. But I think the refactoring steps to come in this ticket "Move the user-restricted tests under the functional/* group and refactor to use an appropriate Feature file" will see these test rewritten anyway, so don't really want to waste time cleaning up too much.